src/amo/reducers/home.js

```diff
@@ -80,14 +106,29 @@ const reducer = (
   action: Action
 ): HomeState => {
   switch (action.type) {
+    case FETCH_HOME_ADDONS:
+      return {
+        ...state,
+        resultsLoaded: false,
+      };
+
     case LOAD_HOME_ADDONS: {
-      const { popularExtensions } = action.payload;
+      const {
+        featuredCollection,
+        popularExtensions,
+      } = action.payload;
 
       return {
         ...state,
+        featuredCollection: featuredCollection.results
+          .slice(0, LANDING_PAGE_ADDON_COUNT)
+          .map((item) => (
+            createInternalAddon(item.addon)
```

Comment:
If this is already on a newline and wrapped in something I'd say typing out return makes this a lot more scannable.

I think this should either be:

```js
.map((item) => createInternalAddon(item.addon))
```
or
```js
.map((item) => {
  return createInternalAddon(item.addon);
})
```

If our lint rules are to blame we should update them; this seems like the worst of both worlds. Though to be clear I prefer the second, more explicit and scannable one 😄